pkg/kafka/policy.go

```diff
@@ -37,6 +51,12 @@ func matchProduceReq(req *proto.ProduceReq, rule api.PortRuleKafka) bool {
 		return false
 	}
 
+	// If both topic and ClientID are specificed,
+	// we need to consider AND relation between them
+	if rule.Topic != "" && rule.ClientID != "" {
```

Comment:
Why is this needed? Shouldn't the code below implement the AND already?

``` go
 	if rule.ClientID != "" && rule.ClientID != req.ClientID {
 		return false
 	}
```

Reply:
ok. Yes, the logic was inverted. My bad. Will remove this.

Reply:
removed